orbit/pkg/table/windowsupdatetable/windowsupdate_test.go

```diff
@@ -14,6 +16,55 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type WindowsUpdatesSearcherMock struct {
+	searchCriteria string
+	SearchResult   *windowsupdate.ISearchResult
+	History        []*windowsupdate.IUpdateHistoryEntry
+}
+
+func (wus *WindowsUpdatesSearcherMock) Search(criteria string) (*windowsupdate.ISearchResult, error) {
+	wus.searchCriteria = criteria
+	return wus.SearchResult, nil
+}
+
+func (wus *WindowsUpdatesSearcherMock) QueryHistoryAll() ([]*windowsupdate.IUpdateHistoryEntry, error) {
+	return wus.History, nil
+}
+
+func TestQueryUpdates(t *testing.T) {
+	t.Run("uses the right criteria is used", func(t *testing.T) {
```

Comment:
nit:
```suggestion
	t.Run("the right criteria is used", func(t *testing.T) {
```

Reply:
Done in b4f731decdf38df5a233fb84cfb6dc5fdafbf740